src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -397,6 +398,43 @@ def get_pixel_size(looks: str) -> float:
     return {'20x4': 80.0, '10x2': 40.0, '5x1': 20.0}[looks]
 
 
+def convert_raster_from_isce2_gdal(input_image, ref_image, output_image):
+    """
+    Convert the water mask in WGS84 to be the same projection and extent of the output product
+    Args:
+        input_image: dem file name
+        ref_image: output geotiff file name
+        output_image: water mask file name
+    """
+    
+    ref_ds = gdal.Open(ref_image)
```

Comment:
Flake8 is complaining about this whitespace.
```suggestion
    """

    ref_ds = gdal.Open(ref_image)
```